backends-velox/src/test/scala/org/apache/gluten/functions/DateFunctionsValidateSuite.scala

```diff
@@ -277,6 +277,36 @@ abstract class DateFunctionsValidateSuite extends FunctionsValidateSuite {
     }
   }
 
+  test("Test to_date function") {
```

Comment:
Nit: simplify the test name to "to_date".

Reply:
This format was used before. Is it appropriate to keep it consistent?

Reply:
Let's just simplify it for new tests. It's spark-test-name style, looks concise.